rudder-web/src/main/scala/com/normation/rudder/web/rest/RestExtractorService.scala

```diff
@@ -624,10 +624,28 @@ case class RestExtractorService (
    * ] }
    */
   def extractNodePropertiesrFromJSON (json : JValue) : Box[RestNode] = {
-    import net.liftweb.json.JsonParser._
-    implicit val formats = DefaultFormats
+    import com.normation.utils.Control.sequence
 
-    Box(json.extractOpt[RestNode]) ?~! "Error when extracting node information"
+    for {
+      props <- json \ "properties" match {
+        case JArray(props) => Full(props)
+        case x             => Failure(s"""Error: the given parameter is not a JSON object with a 'properties' key""")
+      }
+      seq   <- sequence(props) { p =>
+                 p match {
+                   case JObject(JField("name", JString(nameValue)):: JField("value", valueValue) :: Nil) =>
```

Comment:
So no, it's ok, it works as advertised in http://www.rudder-project.org/rudder-api-doc/#api-Nodes-updateNodeProperties
You can remove properties by setting their value to "", which allow to only POST the subset of properties to update in the general case (i.e, it you have 1000 properties and just have one to update, you only give that one in the POST, and the other are not removed but let untouched).